Neos.Flow/Classes/Property/TypeConverter/MediaTypeConverter.php

```diff
@@ -50,7 +50,7 @@ class MediaTypeConverter extends AbstractTypeConverter implements MediaTypeConve
      * @param string $targetType must be "array"
      * @param array $convertedChildProperties
      * @param PropertyMappingConfigurationInterface $configuration
-     * @return array
+     * @return array|string|integer Note that this TypeConverter may return a non-array in case of JSON media type, even though he declares to only convert to array
```

Comment:
Will be "fixed" in master by deprecating this "TypeConverter" and putting something else (a "Serializer"?) in place, that is not restricted to one target type.